eta/core/datasets.py

```diff
@@ -1026,6 +1232,17 @@ def append(self, labeled_data_record):
         '''
         self.index.append(labeled_data_record)
 
+    def cull_with_function(self, func):
```

Comment:
But that's for a different class and does a similar task.  For example, in python, both `set` and `dict` have a method called `update()` which does a similar task, but not exactly the same because the `set` and `dict` classes are different data structures.

Reply:
In other words: [This](https://github.com/voxel51/eta/blob/758d449f258e0e0c13b0b061727fe983cc22db11/eta/core/datasets.py#L1235) is to [this](https://github.com/voxel51/eta/blob/fef974f48c0544374da3da92812eb92161258bba/eta/core/data.py#L967), as [this](https://docs.python.org/3.6/library/stdtypes.html#frozenset.update) is to [this](https://docs.python.org/3.6/library/stdtypes.html#dict.update).